AppController/lib/monit_interface.rb

```diff
@@ -260,5 +260,18 @@ def self.run_cmd(cmd, sleep = false)
     }
     output
   end
+
+  def self.monit_summary
+    summary_dict = {}
+    output = `monit summary`
```

Comment:
As far as I can tell, this implementation keeps master behavior.
I think it's fine to limit a scope of this PR, but it would be nice to review how this monit summary is used later, we may also want to see processes supervised by systemd and adminserver.